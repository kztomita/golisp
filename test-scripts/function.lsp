; system function symbol
(print (function +))

; function symbol
(defun foo ())

(print (function foo))
(print #'foo)

; lambda function
(print (function (lambda (x) (+ x 1))))

; XXX 現状マクロ名を渡してもオブジェクトを返してしまう
