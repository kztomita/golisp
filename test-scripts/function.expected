<system function +>
<function>
<function>
<function>
